src/fetch/fetchSearchHotel.jsx

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios'
+
+const fetchSearchHotel = async ({ queryKey }) => {
+  const keyword = queryKey[1]
```

Comment:
index로 접근하는 것 보단 JSON Object를 활용하여 아래와 유사하게 접근하는건 어떨까요?
```suggestion
  const keyword = queryKey.keyword
```